site/content/en/docs/Guides/Client SDKs/go.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Go Game Server Client SDK"
+linkTitle: "Go"
+date: 2019-05-17T10:17:50Z
```

Comment:
What is this date referring to?

Reply:
The date the page got created, that's it.
https://gohugo.io/content-management/front-matter/

In this case, it's not particularly relevant.